lib/utils/FileCache.mjs

```diff
@@ -69,7 +70,7 @@ export default class FileCache {
 		let changed = false;
```

Comment:
[suggestion] Minor. This assigning seems unnecessary because the `changed` variable is set within the next `if` block;

```suggestion
		let changed;
```